contents/tutorials/variance-connector.md

```diff
@@ -0,0 +1,74 @@
+- **Milestones**: In sales scenarios, it’s common to want to see the progress a specific prospect or customer has made across all of their journey stages. Milestones make it easy to describe those sales stages in a data-driven way and monitor them in real-time on all accounts and contacts.
+- **Real-time alerts and synthetic events**: Another big component of Variance is the ability to alert anyone on the team of exactly the activity they want from only the customers they care about (such as the ones they are marked “owner” of in CRM). In addition, Variance lets teams create “synthetic events” they can be alerted of, such as when a customer is fully onboarded or has reached PQL.
+- **CRM sync**: Finally, a key function of Variance is to make this data easy to sync back to your CRM tools. Unfortunately, those tools aren’t great at understanding event-driven data right now, so Variance reshapes it into a format that makes sense, making it easy to create views or kick off campaigns in your CRM using product data.
+
+If any of that sounds interesting, then follow along as we break down those common use cases...
+
+## Account-level activity
+This is probably the most common use case for Variance, and also one of the simplest. The Variance Connector offers a few different methods for generating accounts based on your product data, the most common of which is to use specific properties as an account name or ID (or both).
+
+When you set up PostHog in Variance, you’ll be asked for your “account matching” method. If you want to explicitly match accounts, you’ll choose the option to use identify with custom traits.
+
+![Variance Connector for PostHog](../images/tutorials/variance/variance-connector-posthog.png)
+
+There you’ll enter any properties you send with your `identify` call that you want to use to generate accounts that are associated with that user. We recommend using `company.name` and `company.id` if possible, but anything is fine. If you don’t have any of that data, you can also just use the domain of the `email` you pass through.
+
+By doing this, we make it easy to see all that activity rolled up on an account profile:
+
+![Variance Connector for PostHog](../images/tutorials/variance/variance-connector-posthog-2.png)
+
+## Milestones
+Milestones are how Variance describes data-driven sales or customer journey stages. Essentially, they’re a partner to the more typical sales stages you would see inside a B2B software organization, but instead of being based on intuition, they’re based on the actual actions of your prospects and customers on your website and in your product. Common Milestones used inside SaaS companies include Onboarded, Activated, and PQL ([product-qualified lead](https://www.variance.com/guides/product-qualified-leads-pql)). 
```

Comment:
```suggestion
Milestones are how Variance describes data-driven sales or customer journey stages. Essentially, they’re a partner to the more typical sales stages you would see inside a B2B software organization, but instead of being based on intuition, they’re based on the actual actions of your prospects and customers on your website and in your product. 

Common Milestones used inside SaaS companies include Onboarded, Activated, and PQL ([product-qualified lead](https://www.variance.com/guides/product-qualified-leads-pql)). 
```